entries/e/ente.io.json

```
@@ -0,0 +1,14 @@
+{
+  "Ente": {
+    "domain": "ente.io",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://ente.io/faq/security-and-privacy/two-factor-support/",
+    "categories": [
+      "backup",
+      "cloud",
+      "security"
```

The `cloud` category is for serverless cloud computing and `security` is for "online security and anti-malware services". For more information, check out the [categories list](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#categories).

```suggestion
      "backup"
```

---

I agree cloud is a mis-tag, but backup also mentions "cross-device file synchronization" which I believe applies to the Ente app

---

Backup is still in the suggestion. I removed `cloud `and `security`.

---

My bad, I misread the diff